src/chapters/vstools/studio.rst

```diff
@@ -0,0 +1,22 @@
+If It Ain't Broke, Add a Breakpoint!
+====================================
+
+On your machine, open up your fork of the ``HelloASPDotNETDemo`` `repository <https://github.com/LaunchCodeEducation/HelloASPDotNETDemo>`_ in Visual Studio.
+The purpose of this studio is to talk about your current debugging strategies and how to make the most of the debugger tools discussed in this chapter.
+
+Before we start practicing with debugging tools, go over with the group one error you encountered when working on your own version of ``HelloASPDotNET``.
+
+#. What was the error?
+#. How did you solve this error? What have been the strategies and tools you have been using so far to debug your code?
+#. Could one of the debugging tools helped you when addressing this error?
+
+Review the code from ``HelloASPDotNETDemo`` and use the debugging tools in Visual Studio to address potentional problems.
+
+Questions to review as you look through the code:
+
+#. When would you use the *Call Stack* pane? If you run the app and it is already functioning, what shows up there? 
+#. What breakpoints should you add?
+#. Would a conditional breakpoint make sense to use in the context of this app?
+
+Once you have gone through the demo code, open up a piece of code you have been struggling with.
```

Comment:
this is a good prompt to get them to think about other programs that they might be struggling with

Reply:
Thank you! I am hoping that it will divert students who want to work on other stuff back to thinking about debugging by giving them the space to work on other stuff.